servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-queue/README.md

```diff
@@ -1,4 +1,4 @@
-# Sample for Spring JMS with Azure Service Bus Queue Spring Cloud client library for Java 
+# Sample for Spring JMS with Azure Service Bus Queue Spring Cloud
```

Comment:
We can remove `Spring Cloud` now as well.

Reply:
Same here.